### ISSUE TITLE
Jumpless Check Suggestion Doesn't Function Properly

### ISSUE THREAD

**Issue by stllr927** [Phantom5800/pmr-tracker#73]
**Describe the bug**
The tracker's handy "check-map" (thank you for this by the way) sees no difference between if I have the ability to jump or not.

**To Reproduce**
1. Go to 'the tracker website'
2. Right-Click on 'Boots'
3. Open the Toad Town "Below Plaza" Map
4. Note that Bub-ulb is considered a "valid" check despite the inability to get up there Boot-less (not the only example but the easiest to show)

**Expected behavior**
I expected it to remove checks that require the ability to jump if I do not have boots.

**Screenshots**
![image](https://user-images.githubusercontent.com/22407114/197640645-0e07e0f9-d900-4694-a6c3-b1bf5c91e159.png)

**Environment:**
 - Browser: Opera GX

**Comment by stllr927** [comment]
Ah, excuse me! I found out recently that "jumpless" is a very new feature to the rando and the tracker alike. Please don't feel the need to rush to fix this.

**Comment by MythicFrog** [comment]
As a note, there are quite a few checks available jumpless with Parakarry. Had a seed start me in Goomba village with Parakarry and was able to make it to Toad Town jumpless so I'm sure there are quite a places accessible in that case.

**Comment by stllr927** [comment]
This is true, there are quite a few accessible with Parakarry. Just not all of them, especially the Mt. Rugged/Boo's Mansion checks.
